src/server/game/Globals/ObjectMgr.cpp

```diff
@@ -2636,8 +2636,10 @@ void ObjectMgr::LoadItemTemplates()
         return;
     }
 
-    _itemTemplateStore.rehash(result->GetRowCount());
+    _itemTemplateStore.reserve(result->GetRowCount());
```

Comment:
before i got people wanting to bitch and complain about this without any or no explaination besides "why?". i advise you to look up what they first, and the reason why i changed it was due to it crashing.

Reply:
Why? 💋

Reply:
🖕